app/code/Magento/Customer/Test/Mftf/Test/StorefrontCreateCustomerAutocompleteTest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCreateCustomerAutocompleteTest">
+        <annotations>
+            <features value="Customer"/>
+            <stories value="Autocomplete to user area input fields"/>
```

Comment:
"As a Store Customer I should be able to use browser autocomplete feature for Customer registration"

Reply:
Updated.
Is correctly?